packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -26,18 +30,28 @@ const AuthDefault = ({
   isV2,
 }: AuthDefaultProps): ReactElement => {
   const [shouldLogin, setShouldLogin] = useState(isV2);
+  const { mutateAsync: checkEmail } = useMutation((emailParam: string) =>
+    request(`${apiUrl}/graphql`, QUERY_USER_BY_EMAIL, { email: emailParam }),
+  );
```

Comment:
I'm not really sure we should do this.
Think Kratos should have a pre-hook for this and do it their side.

Reply:
Totally agree. What do you think of commenting this block out for the meantime, so we can push forward with what we have here?